scripts/docker/requirements_dev.txt

```diff
@@ -0,0 +1,5 @@
+sudo
```

Comment:
Damn `sudo` is a requirement?

Also, can we add `vim`?

Reply:
added

Reply:
Now I know why vim was in my docker container LOL!